boa/src/builtins/json.rs

```diff
@@ -23,20 +22,39 @@ pub fn parse(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
 /// Process a Javascript object into a JSON string
 pub fn stringify(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
     let obj = args.get(0).expect("cannot get argument for JSON.stringify");
-    let json = obj.to_json();
-    Ok(to_value(to_string_pretty(&json).expect("")))
+    let json: String = obj.to_json().to_string();
```

Comment:
Doesn't type inference work here?
```suggestion
    let json = obj.to_json().to_string();
```